tools/packages/minion/minion.spec

```diff
@@ -155,9 +155,12 @@ rm -rf %{buildroot}%{minioninstprefix}/demos
 
 # Copy over the run script
 mkdir -p %{buildroot}%{_initrddir}
-sed -e 's,@INSTPREFIX@,%{minioninstprefix},g' %{_builddir}/%{_name}-%{version}-%{release}/tools/packages/minion/minion.init > "%{buildroot}%{_initrddir}"/minion
+sed -e 's,@INSTPREFIX@,%{minioninstprefix},g' -e 's,@SYSCONFDIR@,%{_sysconfdir}/sysconfig,g'  %{_builddir}/%{_name}-%{version}-%{release}/tools/packages/minion/minion.init > "%{buildroot}%{_initrddir}"/minion
```

Comment:
ooh i didn't know you could pass two scripts on the same sed command line, neato :)